app/src/main/java/com/jnj/vaccinetracker/common/data/models/Constants.kt

```diff
@@ -52,4 +52,6 @@ object Constants {
 
     // PROGRESS
     const val MAX_PERCENT = 100
+
+    const val REQ_REGISTER_PARTICIPANT = 453
```

Comment:
what does this const exactly mean? from name I'm not able to determine meaning of this const

Reply:
The problem is I copied the name from the existing implementation. Not sure what it means

Reply:
I just moved it from the class to constants to make a single source of truth.